tests/stdlib/tos.nim

```diff
@@ -664,32 +664,6 @@ block: # normalizePathEnd
     doAssert r"E:/".normalizePathEnd(trailingSep = true) == r"E:\"
     doAssert "/".normalizePathEnd == r"\"
 
-block: # isValidFilename
```

Comment:
bring these tests back too.

Reply:
Yes, I want to, the CI was always timing out after ~100 minutes, I think Gitlab was down?.